lib/spack/spack/test/concretize.py

```diff
@@ -22,6 +22,9 @@
 from spack.util.mock_package import MockPackageMultiRepo
 from spack.version import ver
 
+pytestmark = pytest.mark.skipif(sys.platform == "win32",
+                                reason="does not run on windows")
+
```

Comment:
I think we'll want to get some of the more-basic tests in here passing: concretization is fundamental.

Reply:
Ok, this test suite blew up quite badly, so I'll look into getting it up and running.

Reply:
Resolved as discussed today during the Spack on Windows meeting.